src/main/java/com/tstepnik/planner/service/StatisticsService.java

```diff
@@ -42,8 +43,8 @@ public Statistics createSaveAndReturnStatistics() {
         return statistics;
     }
     
-    private boolean todayWasNotSavedStatistics(LocalDateTime creationDate) {
-        LocalDateTime now = LocalDateTime.now();
+    private boolean isBeforeToday(LocalDateTime creationDate) {
+        LocalDate now = LocalDate.now();
 
         return creationDate.getDayOfYear() != now.getDayOfYear() &&
```

Comment:
https://www.geeksforgeeks.org/localdate-isbefore-method-in-java-with-examples/

Reply:
Thanks, I had to convert LocalDateTime to LocalDate first.